morphsnakes.py

```diff
@@ -202,6 +202,55 @@ def circle_level_set(image_shape, center=None, radius=None):
     return res
 
 
+def ellipsoid_level_set(image_shape, center=None, scale_factor=0.9):
```

Comment:
The function name is misleading. As a user of a function with this name, I would expect it to give me the possibility to set the location of the ellipse, its eccentricity or the lengths of the major and minor axis, and ideally also an angle.

I think that this function is very useful, but it is more related to `circle_level_set` than to an ellipse. I think we can incorporate this function inside `circle_level_set` by adding a new parameter in `circle_level_set` (for example, `aspect_ratio`) that specifies the aspect ratio of the circle. If it is set to `'image'`, it will get the aspect ratio of the image, effectively reproducing this behavior.

Reply:
I agree that I would be better to have an option for adapting the level set to the shape of the image if dimensions are different. In that case, should the `radius` parameter be adapted as well?